server/src/test/java/org/elasticsearch/index/query/IntervalQueryBuilderTests.java

```diff
@@ -0,0 +1,173 @@
+package org.elasticsearch.index.query;
+
+import org.apache.lucene.search.Query;
+import org.apache.lucene.search.intervals.IntervalQuery;
+import org.apache.lucene.search.intervals.Intervals;
+import org.elasticsearch.search.internal.SearchContext;
+import org.elasticsearch.test.AbstractQueryTestCase;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.instanceOf;
+
+public class IntervalQueryBuilderTests extends AbstractQueryTestCase<IntervalQueryBuilder> {
+
+    @Override
+    protected IntervalQueryBuilder doCreateTestQueryBuilder() {
+        assumeTrue("test runs only when at least a type is registered", getCurrentTypes().length > 0);
```

Comment:
You don't need this anymore, this method always run with a single registered type.